packages/storage-mixin/package.json

```diff
@@ -42,7 +42,6 @@
     "@mongodb-js/compass-logging": "^1.1.4",
     "ampersand-model": "^8.0.1",
     "ampersand-rest-collection": "^6.0.0",
-    "ampersand-sync": "^5.1.0",
```

Comment:
Not that it doesn't make sense to remove this, but just FYI `ampersand-model` and `ampersand-rest-collection` both have a dependency on this package anyway